src/test/resources/sample2query.json

```diff
@@ -0,0 +1,11 @@
+{
+    "workspace" : "src/test/resources/tdbworkspace",
+    "array" : "sample2Array",
+    "query_column_ranges" : [ [ [0, 17421565 ] ] ],
+    "vid_mapping_file": "src/test/resources/sample2vid.json",
+    "callset_mapping_file": "src/test/resources/sample2callsets.json",	
+    "query_attributes" : [],
+    "reference_genome" : "/seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta",
+    "vcf_header_filename" : "src/test/resources/sample2header.vcf",
```

Comment:
Is this required by genomicsdb? It really sucks that we have to duplicate the sample2 header information in two places.